### PR TITLE
Create purging checkpointer before initial replication.

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

medic/cht-core#7419

# Code review checklist

- [x] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [x] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [x] Tested: Unit and/or e2e where appropriate
- [x] Internationalised: All user facing text
- [x] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @latin-panda . Would you mind a review? Thanks!

File: webapp/src/js/bootstrapper/index.js [hunk 1]
```diff
@@ -102,14 +102,19 @@
...
+
   const initialReplication = function(localDb, remoteDb) {
     setUiStatus('LOAD_APP');
     const dbSyncStartTime = Date.now();
     const dbSyncStartData = getDataUsage();
 
-    return purger
-      .info()
-      .then(info => {
+    return createPurgingCheckpoint()
```
Comment (latin-panda) [inline webapp/src/js/bootstrapper/index.js:114]: What do you think if `createPurgingCheckpoint` is in the bootstrapper's [main function](https://github.com/medic/cht-core/blob/b2314e7f865a245a5ab5aa141d1903cc51fcf562/webapp/src/js/bootstrapper/index.js#L234) and have this function to only be about replication?

The main function is already big but still easier to see the step by step process: setting purging options, creating checkpoint (can these 2 be on the same function? ex: `initPurge`), checking if purge should happen, etc..

**Review by latin-panda — COMMENTED** [review]
Nice e2e! ✨✨ I left a comment inline, let me know your thoughts.

File: webapp/src/js/bootstrapper/index.js (hunk 1 above)
Comment (dianabarsan) [reply webapp/src/js/bootstrapper/index.js:114]: We only want to create the checkpoint when doing initial replication, so I think it belongs in this function,

**Review by latin-panda — APPROVED** [review]
This looks great! ✨

**Pull request merged**